string msg_type
uint32 robot_id
uint32 desire_position
uint32 talk_to          #use robot id
